content/en/hosting-and-deployment/hosting-on-21yunbox.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Host on 21YunBox
+linktitle: Host on 21YunBox
+description: Host your Hugo site with 21YunBox's blazing fast Chinese CDN, fully-managed SSL and auto deploys from Gitee.
+date: 2021-01-06
+publishdate: 2021-01-06
+lastmod: 2021-01-06
+categories: [hosting and deployment]
+keywords: [21yunbox,hosting,deployment]
+authors: [Toby Glei]
+menu:
+  docs:
+    parent: "hosting-and-deployment"
+    weight: 10
+weight: 10
+sections_weight: 10
+draft: false
+aliases: []
+toc: true
+---
+
+## Introduction
+
```

Comment:
Can you remove the Introduction section.

Reply:
removed.